test/test_model.py

```diff
@@ -1,49 +1,85 @@
+from typing import Tuple
+
 import pytest
 from pytest import fixture
 import torch
 import numpy as np
+
+from config import Config
 from model import Model
+from src.utils import load_data
+
 
 @fixture
 def input():
     np.random.seed(42)
     data = np.random.uniform(0, 10, 24).reshape(24, 1)
     return torch.Tensor(data)
 
+@fixture 
```

Comment:
Probably should create a contest module for fixtures that span modules so the module test code can exist in independent test modules... Can be on the TODO list.